apps/frontend/src/modules/onboarding/slices/actions.ts

```diff
@@ -1,10 +1,24 @@
 import { createAsyncThunk } from "@reduxjs/toolkit";
 
 import { type AsyncThunkConfig } from "~/libs/types/types.js";
-import { type OnboardingGetAllResponseDto } from "~/modules/onboarding/onboarding.js";
+import {
+	type OnboardingAnswerRequestDto,
+	type OnboardingGetAllResponseDto,
+	type OnboardingUserAnswerDto,
+} from "~/modules/onboarding/onboarding.js";
 
 import { name as sliceName } from "./onboarding.slice.js";
 
+const saveAnswers = createAsyncThunk<
+	OnboardingUserAnswerDto[],
+	OnboardingAnswerRequestDto,
+	AsyncThunkConfig
+>(`${sliceName}/create-user`, async (createAnswerPayload, { extra }) => {
```

Comment:
why does path of this action contains `/create-user`? Maybe `/save-answers` would be more suitable.